paper-checkbox.html

```diff
@@ -136,6 +138,17 @@
       }
     },
 
+    // button-behavior hook
+    _buttonStateChanged: function() {
+      this.checked = this.active;
+    },
+
+    _checkedChanged: function(checked) {
+      this.setAttribute('aria-checked', this.checked ? 'true' : 'false');
+      this.active = this.checked;
```

Comment:
I think so. This is also how @sjmiles added the behaviour to `paper-toggle-button`. 

I think normally `active` would be enough, but we (the internet) are used to setting `checked` on a checkbox, so it's basically making sure that checked iff active.